### DISCUSSION TITLE
/me chat colour is very difficult to read and inaccessible

### DISCUSSION THREAD

**Discussion by madrxx** [EssentialsX/Essentials#5020] [Ideas and Feature Suggestions]
![image](https://user-images.githubusercontent.com/74277456/181996853-5bd538f2-62ab-4480-b7eb-53ed4771ad14.png)

The purple colour used in Essentials' /me command is very dark and difficult to read, and according to the server owner not configurable (if it is somehow, a better default should still be set).

**Comment by pop4959** [comment]
This is configurable via a custom message file. You can learn how to set that up [here](https://essentialsx.net/wiki/Locale.html). For the message in your screenshot take a look at changing [`action`](https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/main/resources/messages.properties#L5).
